azerothcore/mod-congrats-on-level
# feat. Add option to Learn or cast spells

Co-authored-by: Ericks Oliveira <ericksoliveira258@gmail.com>

## Changes Proposed:
- Currently, the module only allows casting spells. But in case some group of people want those spells to be permanent, then they need the learning option.
- With this new attribute in the database, you can now choose whether you want to cast or learn the spell. By default, it is only cast, but if you want you can change it.

## Issues Addressed:

- Closes https://github.com/azerothcore/mod-congrats-on-level/pull/29
- Closes https://github.com/azerothcore/mod-congrats-on-level/issues/28

## SOURCE:


## Tests Performed:

- Windows 10
- Ingame

## How to Test the Changes:


1. Create a character.
2. Add a new award or edit any of the existing ones.
3. Configure a spell, so that it is by cast and another by learning.
4. Depending on the option used, one should be permanent and the other temporary.
